README.md

```diff
@@ -34,3 +34,5 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+Test
```

Comment:
Do we need this here?

Reply:
I agree we can remove this. I have updated it in this SHA https://github.com/dustinsoos/bored-user/pull/1/commits/c37d56026dce1e1230588e6b741861f6e9f9880f